servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ConnectionInfo.java

```diff
@@ -39,6 +39,16 @@
      */
     SocketAddress remoteAddress();
 
+    /**
+     * Get the {@link SslConfig} for this connection.
+     *
+     * @return The {@link SslConfig} if SSL/TLS is configured, or {@code null} otherwise.
+     */
+    @Nullable
+    default SslConfig sslConfig() { // FIXME: 0.43 - consider removing default impl
```

Comment:
Added more reviewers for visibility of this message:

WDYT about exposing `sslConfig()` through `ExecutionContext` instead of `ConnectionInfo`? That way it will be automatically visible through `client.executionContext()` and `serverContext.executionContext()` too.

Reply:
We discussed this offline, and agreed that its more relevant as part of the `ConnectionInfo`.

Reply:
For visibility: client is an abstraction that does not know about the actual connections. There are implementations of the client (multi-address client) that can talk to HTTP and HTTPS targets. Therefore, having `SslConfig` at the client level does not make sense.
We can add `serverContext.sslConfig()` later, if we have a use-case for that. For now, will just keep it at `ConnectionInfo`.